src/Feed/FeedManager.php

```diff
@@ -7,31 +7,24 @@
  */
 class FeedManager
 {
-    /**
-     * @var array
-     */
-    protected $feeds = [];
+    protected array $feeds = [];
 
     /**
      * Register a feed in the manager.
```

Comment:
Can be removed I guess

Reply:
What do you want to remove ?

Reply:
The comment "Register a feed in the manager."